.github/workflows/build-and-deploy-prod.yml

```diff
@@ -42,7 +42,8 @@ jobs:
                 VITE_PROJECT_NAME: ${{ vars.VITE_PROJECT_NAME }}
                 VITE_CROWDIN_BRANCH_NAME: ${{ vars.VITE_CROWDIN_BRANCH_NAME }}
                 VITE_TRANSLATIONS_CDN_URL: ${{ vars.VITE_TRANSLATIONS_CDN_URL }}
-
+                VITE_TRACKJS_TOKEN: ${{ vars.VITE_TRACKJS_TOKEN }}
```

Comment:
@farrah-deriv would you please help to add the trackjs token to the github variables?